docs/docs/widgets/text.md

```diff
@@ -76,29 +72,29 @@ Makes the component visible in mobile view. You can set it with the toggle butto
 
 | Text Property          | Description                           | Configuration Options |
 |:--------------------|:--------------------------------------|:---------------|
-| Size         | Dimensions of the characters in a font. | Enter any number between `1-100` or dynamically configure it using `fx` |
-| Weight       | Determines how bold or light your text will appear. | Select from `light`, `regular`, `semi-bold` or `bold` or dynamically configure it using `fx` |
-| Style        | Allows you to apply styles like italic or normal, altering the overall look of the text content. | Select from `normal`, `italic` or `oblique` or dynamically configure it using `fx` |
-| Color        | Sets the color of the text. | Choose a color using the color picker or dynamically configure it using `fx` |
-| Scroll        | Creates a scroll bar if the text exceeds the component's dimensions. | Choose between `enable` or `disable` or dynamically configure it using `fx` |
-| Line Height       | Determines the vertical space between lines of text within an element. | Enter a number as the value (example: `1.5`) or dynamically configure it using `fx`|
-| Text Indent       | Commonly used to create an indentation effect. | Enter a number as the value (example: `10`) or dynamically configure it using `fx`|
-| Alignment        | Sets the alignment of the text. | Select the available options to align the text vertically or horizontally or dynamically configure it using `fx`|
-| Text Decoration   | Adds an underline, overline, line-through, or a combination of lines to selected text. | Select one of the available options - `none(default)`, `underline`, `overline`, and `strike-through` or dynamically configure it using `fx` |
-| Transformation | Dictates the capitalization of text. | Select one of the available options - `none (default)`, `uppercase`, `lowercase`, `capitalize` or dynamically configure it using `fx`|
-| Letter spacing | Determines the space between each letter. | Enter a number as the value (example: `15`) or dynamically configure it using `fx`|
-| Word spacing | Determines the space between each word. | Enter a number as the value (example: `15`) or dynamically configure it using `fx`|
-| Font variant | Adjusts the text appearance by applying font variations. | Select one of the available options - `normal`, `inherit`, `small-caps`, `initial` or dynamically configure it using `fx`|
+| Size         | Dimensions of the characters in a font. | Enter any number between `1-100` or dynamically configure it using **fx** |
```

Comment:
can you please add full stops at the end of these sentences, also wherever applicable in the doc?